components/mpas-framework/Makefile

```diff
@@ -700,6 +703,25 @@ ifeq "$(OPENMP_OFFLOAD)" "true"
 	LDFLAGS += $(LDFLAGS_GPU)
 endif #OPENMP_OFFLOAD IF
 
+ifeq "$(USE_MAGMA)" "true"
+   LIBS += -L$(MAGMADIR)/lib -lmagma
+   FCINCLUDES += -I${MAGMADIR}/include
+   override CPPFLAGS += "-DUSE_MAGMA"
+   override FFLAGS_ACC += -Dmagma_devptr_t='integer(kind=8)' -Mcudalib=cublas
```

Comment:
Interesting to see `-Dmagma_devptr_t='integer(kind=8)` in FFLAGS. What's the story behind it?

Reply:
@sarats , I was following MAGMA's instruction, but I just tried to compile without the flag and use 'integer(kind=8)' in the code directly, and it works. It'd be better to reduce complexity in the code and Makefile, so I will remove the flag!